packages/server/src/blob/create.rs

```diff
@@ -93,13 +94,13 @@ impl Server {
 						let p = transaction.p();
 						let statement = formatdoc!(
 							"
-								insert into objects (id, bytes, complete, count, weight, touched_at)
-								values ({p}1, {p}2, {p}3, {p}4, {p}5, {p}6)
-								on conflict (id) do update set touched_at = {p}6;
+								insert into objects (id, bytes, complete, count, depth, weight, touched_at)
+								values ({p}1, {p}2, {p}3, {p}4, {p}5, {p}6, {p}7)
+								on conflict (id) do update set touched_at = {p}7;
 							"
 						);
 						let now = time::OffsetDateTime::now_utc().format(&Rfc3339).unwrap();
-						let params = db::params![&id, &data.bytes, 1, 1, size, now];
+						let params = db::params![&id, &data.bytes, 1, 1, 0, size, now];
```

Comment:
Isn't the depth of a leaf supposed to be 1, not 0?

Reply:
Fixed.